docs: :memo: add content to website landing page (#77)
rostools/github-intro: index.qmd

# Welcome!

This 3-hour workshop is a gentle introduction to using GitHub for

> managing and working with files. It is designed for doing participatory
> live-"typing", where the instructor does the activity on their computer
> that is on the projector and learners type along with what is done.
> Along with the live-typing, there are reading tasks, discussion
> activities, and hands-on practical exercises interspersed throughout the
> workshop.

Reviewer: ```suggestion
managing and working with files. It is designed for doing participatory
live-typing, where the instructor demonstrates the tasks on their computer
connected to a projector while learners type along on their own computer.
The workshop also includes reading tasks, discussion
activities, and hands-on exercises.
```

> This website contains all of the material for the workshop, from reading
> material to exercises to presentations to live-typing material to
> images. It is structured as a book, with "chapters" as sessions, given

Reviewer: ```suggestion
This website contains all of the material for the workshop, including readings, exercises, presentations, live-typing material, and
images. It is structured as a book, with "chapters" as sessions,
```

in order of appearance. We make heavy use of the website throughout the
course where "type-along" sessions almost identically follow the
material on the website (with slight modifications for time or more
detailed explanations).

The workshop material is created using [Quarto](https://quarto.org) to
write the material and create the book format,
[GitHub](https://github.com/) to host the [Git](https://git-scm.com/)
repository of the material, and [GitHub
Actions](https://github.com/features/actions) with

> [Netlify](https://www.netlify.com/) to build and host the website. The

Reviewer: Is this info necessary here? Could maybe be further down in its own section?

original source material for this course is found on the
[`github-intro`](https://github.com/rostools/github-intro) GitHub
repository.

Want to contribute to this course? Check out the
[README](https://github.com/rostools/github-intro/blob/main/README.md)
file as well as the
[CONTRIBUTING](https://github.com/rostools/github-intro/blob/main/CONTRIBUTING.md)
file on the GitHub repository for more details. The easiest way to
contribute is by using [GitHub](https://github.com/) and creating a [new
issue](https://github.com/rostools/github-intro/issues/new) to make
comments and give feedback on the material.

<!-- TODO: Convert this into an includes snippet? -->

::: {.callout-tip appearance="minimal"}
:star: Do you find this workshop material useful? Please consider
"starring" our [GitHub
repository](https://github.com/rostools/github-intro/) to show your

> support! Starring will save it to a list of repositories you like that
> you can easily find again, plus it helps give our project more
> visibility!

Reviewer: ```suggestion
appreciation! Starring will save it to your list of saved repositories, so it’s easy to find again later. As a plus, it helps give our project more
visibility! :star2:
```

> :::

Reviewer: 🌟 🌟 🌟

## Target audiences

This website and its content are targeted to three groups:

1.  For the **learners** to use during the course, both to follow along
    in case they get lost and also to use as a reference after the

>     workshop ends. The learner is someone who is currently or will soon
>     actively be doing research (e.g. a PhD or postdoc) and who has no or
>     little knowledge of using Git and GitHub. A more detailed

Reviewer: ```suggestion
    workshop ends. A more detailed
```
Maybe fine to not include descriptions of the learner here by refer to the Is this for you section instead where that info should be included already (i.e., minimise duplicate content)

    description of who the learner is can be found in
    @sec-is-it-for-you.
2.  For the **instructors** to use as a guide for when they do the
    type-along sessions and lectures.
3.  For those who are **interested in teaching**, who may not have much
    experience or may not know where to start, to use this website as a
    guide to running and instructing their own workshops.

## Re-use and licensing

> <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img src="https://i.creativecommons.org/l/by/4.0/80x15.png" alt="Creative Commons License" style="border-width:0"/></a>

Reviewer: Damn, that is not a pretty badge. Is 
![](https://licensebuttons.net/l/by/4.0/88x31.png) less ugly?

For reference, the current badge looks like this: 
![](https://i.creativecommons.org/l/by/4.0/80x15.png)

Author: Or just remove it. We have it in the page footer anyway.

> 
> <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img src="https://i.creativecommons.org/l/by/4.0/80x15.png" alt="Creative Commons License" style="border-width:0"/></a>

Author: ```suggestion

```

> This 3-hour workshop is a gentle introduction to using GitHub for

Reviewer: ```suggestion
This 3-hour workshop is a gentle introduction to using [GitHub](https://github.com/) for
```

1.  For the **learners** to use during the course, both to follow along
    in case they get lost and also to use as a reference after the
    workshop ends. A more detailed description of who the learner is can

>     be found in @sec-is-it-for-you.

Reviewer: ```suggestion
    be found the [Is this for you?](/preamble/syllabus.qmd#sec-is-it-for-you) section.
```
This might be a more general thing, but the `@` renders to “Section 2.2” which is less informative than the title itself :)

Author: Hmmm.. But then you have to remember the title, and if it changes, you have to change it too. Hmm, I'll think about this one before accepting. Maybe there's another compromise.

Reviewer: Yeahhhh, I see your point. Let’s leave it as it for now and think about if there’s another solution.

[CONTRIBUTING](https://github.com/rostools/github-intro/blob/main/CONTRIBUTING.md)
guidelines and the [For Instructors](/appendix/for-instructors.qmd)
section for more details and tips on using this material.

## Contributing

Want to contribute to this course? Check out the
[README](https://github.com/rostools/github-intro/blob/main/README.md)
file as well as the
[CONTRIBUTING](https://github.com/rostools/github-intro/blob/main/CONTRIBUTING.md)
file on the GitHub repository for more details. The easiest way to

> contribute is by using [GitHub](https://github.com/) and creating a [new
> issue](https://github.com/rostools/github-intro/issues/new) to make
> comments and give feedback on the material.

Reviewer: ```suggestion
contribute is by creating a [GitHub
issue](https://github.com/rostools/github-intro/issues/new) to make
comments and give feedback on the material.
```

> Want to contribute to this course? Check out the
> [README](https://github.com/rostools/github-intro/blob/main/README.md)
> file as well as the
> [CONTRIBUTING](https://github.com/rostools/github-intro/blob/main/CONTRIBUTING.md)
> file on the GitHub repository for more details. The easiest way to
> contribute is by using [GitHub](https://github.com/) and creating a [new
> issue](https://github.com/rostools/github-intro/issues/new) to make
> comments and give feedback on the material.

Reviewer: ```suggestion
Want to contribute to this course? The easiest way to
contribute is by creating a [GitHub
issue](https://github.com/rostools/github-intro/issues/new) to make
comments and give feedback on the material. Make sure you check out the
[README](https://github.com/rostools/github-intro/blob/main/README.md)
file and the
[CONTRIBUTING](https://github.com/rostools/github-intro/blob/main/CONTRIBUTING.md)
guide for more details.
```
Just a little rearranging to make it feel a bit “easier” to contribute.
